### ISSUE TITLE
A11y_SQL Azure Data Studio_Database_Create project from database_Voiceover:  voiceover is not announcing required to "server" connect control.

### ISSUE THREAD

**Issue by ShaileshChudawkar** [microsoft/azuredatastudio#24344]
### GitHub Tags:
#A11yMAS; #A11ySev3; #A11yTCS; #Mac #BM_SQL Azure Data Studio_Mac_Sept2023; #SQL Azure Data Studio; #WCAG1.3.1; #VoiceOver; #Benchmark;

### Environment Details:
Application Name: SQL Azure Data studio
OS: Mac
### Repro steps:

1. Launch the application-Azure Database Project and install extension SQL Database Projects.
2. Navigate to SQL Database Project Home Screen with heading "Projects".
3. Tab to connections select your server and connect to it.
4. Few databases will appear under your connection then right click on any database
5. Tab till "Create project from database" and hit enter.
6. Verify voiceover is not announcing required to "server" connect control.
### Actual Result:
Voiceover is not announcing required field to the "server" connect control.
Expected Result: 
Voice over should announce required to the "server" connect control.
### User Impact:
Screen reader users face difficulties if voiceover is not announcing required to required field
### Attachment:

https://github.com/microsoft/azuredatastudio/assets/143179527/c18a74b9-555d-4dc8-8173-7c17f4ef0fa8
